clinica/iotools/converters/ukb_to_bids/ukb_utils.py

```diff
@@ -0,0 +1,311 @@
+from os import PathLike
+from typing import BinaryIO, Iterable, List, Tuple, Union
+
+from pandas import DataFrame
+
+
+def find_clinical_data(
+    clinical_data_directory: PathLike,
+) -> Tuple[DataFrame, DataFrame, DataFrame, DataFrame, DataFrame]:
+    from pathlib import Path
+
+    import pandas as pd
+
+    from clinica.utils.stream import cprint
+
+    # Read the xls
+    try:
+        image_data_file = list(Path(clinical_data_directory).glob("*.csv"))
+    except StopIteration:
+        raise FileNotFoundError("Imaging collection file not found.")
```

Comment:
Shouldn't it be clinical data files not found?

Reply:
Yes indeed.